### PR TITLE
fix: Init target views

### PR DESCRIPTION
**Pull request by adityachoudhari26**

### CODE REVIEW

File: apps/webservice/src/app/[workspaceSlug]/_components/target-condition/TargetConditionDialog.tsx [hunk 1]
```diff
@@ -94,3 +114,190 @@ export const TargetConditionDialog: React.FC<TargetConditionDialogProps> = ({
...
+      .then(() => router.refresh());
+  };
+
+  return (
+    <Dialog
+      open={open}
+      onOpenChange={(open) => {
+        setOpen(open);
+        if (!open) onClose?.();
+      }}
```
Comment (zacharyblasczyk) [inline apps/webservice/src/app/[workspaceSlug]/_components/target-condition/TargetConditionDialog.tsx:286]: is this right?

Comment (adityachoudhari26) [reply apps/webservice/src/app/[workspaceSlug]/_components/target-condition/TargetConditionDialog.tsx:286]: yes

File: packages/db/src/schema/target.ts [hunk 2]
```diff
@@ -75,6 +77,26 @@ export const targetSchema = pgTable(
...
+  filter: jsonb("filter").notNull().$type<TargetCondition>(),
+});
+
+export type TargetView = InferSelectModel<typeof targetView>;
+
+export const createTargetView = createInsertSchema(targetView, {
+  filter: targetCondition,
+}).omit({
+  id: true,
+});
```
Comment (jsbroks) [inline packages/db/src/schema/target.ts:96]:
```suggestion
}).omit({ id: true });
```

File: apps/webservice/src/app/[workspaceSlug]/(targets)/layout.tsx [hunk 3]
```diff
@@ -38,6 +44,20 @@ export default function TargetLayout({
...
+  /**
+   * Views is a 2016 album by Drake that showcases his blend of rap and R&B
+   * with influences from his hometown, Toronto. It features hit singles like
+   * "Hotline Bling" and "One Dance," reflecting Drake's introspective lyrics
+   * on fame, relationships, and his rise to the top. The album explores
+   * themes of loyalty and love, all while solidifying his position as one of
+   * the most influential artists in contemporary hip-hop.
+   *
+   * This is NOT a reference to that album.
+   */
```
Comment (zacharyblasczyk) [inline apps/webservice/src/app/[workspaceSlug]/(targets)/layout.tsx:57]: ❤️

**Review by jsbroks — APPROVED** [review]

**Pull request merged**